suclass/cms/templates/body-post.html

```diff
@@ -1 +1,12 @@
-<script id="ze-snippet" src="https://static.zdassets.com/ekr/snippet.js?key=8ca24832-80a4-4e66-ba71-c44c36c96b85"></script>
+<script>
+	$(document).ready(
+		setTimeout(
+			function()
+			{
+				var s = document.createElement("script");
+				s.id = "ze-snippet"
```

Comment:
In languages which allow single and double quotation marks to be used interchangeably (here, JavaScript and Python), we opt for single quotes, by default, for consistency.

The exceptions, instances where we might opt for double quotes instead are:
- interpolated strings (in python)
  - `result = "foo {bar}".format(bar='baz')`
    - This follows the POSIX shell convention of only allowing variable interpolation into double-quoted strings.
- strings with many embedded single quotes"
  - `result = "It's painful if we're needin' to escape lots 'o quotes!"`
  - `result = 'It\'s painful if we\'re needin\' to escape lots \'o quotes!'`

Given that, let's switch all of the double-quotes here into singles.